twindb_backup/destination/ssh.py

```diff
@@ -1,31 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Module for SSH destination.
+"""
 import base64
-from contextlib import contextmanager
 import json
 import os
 import socket
 from subprocess import Popen, PIPE
-from twindb_backup import log
+
+from twindb_backup import LOG
 from twindb_backup.destination.base_destination import BaseDestination, \
     DestinationError
+from twindb_backup.util import run_command
 
 
-class Ssh(BaseDestination):
+class SshConnectInfo(object):  # pylint: disable=too-few-public-methods
```

Comment:
What is the benefit of having this as a separate class if this is only ever used in the `Ssh` class.